lib/maths/unittest/CDecayRateControllerTest.cc

```diff
@@ -109,4 +108,47 @@ BOOST_AUTO_TEST_CASE(testPersist) {
     }
 }
 
+BOOST_AUTO_TEST_CASE(testBehaviourAfterPersistAndRestore) {
```

Comment:
This will need to be a CppUnit test in 6.8, with a method declaration added to the header too.